apiserver/facades/controller/caasapplicationprovisioner/provisioner.go

```diff
@@ -310,13 +302,13 @@ func (a *API) garbageCollectOneApplication(args params.CAASApplicationGarbageCol
 	if err != nil {
 		return errors.Trace(err)
 	}
-	observedUnitTags := set.NewTags()
+	observedUnitTags := set.NewStrings()
```

Comment:
Why change from tags to strings?

Reply:
I don't know, foundUnits changed as well

Reply:
@hpidcock do you know why this was changed?